app/models/competition.rb

```diff
@@ -2847,7 +2858,6 @@ def self.form_data_json_schema
           "type" => "object",
           "properties" => {
             "isConfirmed" => { "type" => "boolean" },
-            "isVisible" => { "type" => "boolean" },
```

Comment:
This line deletion seems unrelated, although I'm aware that we're probably not really using it anywhere